.github/workflows/main.yml

```diff
@@ -126,7 +126,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        version: ['v2.2.6', 'v2.3.3', 'v2.4.5', 'v2.4.6', 'v2.5.0']
+        version: ['v2.2.6', 'v2.3.3', 'v2.4.6', 'v2.5.0', 'v2.5.1']
```

Comment:
Think it makes sense to stick to your old scheme and just test 2.5.1 instead of 2.5.0.
2.4.5/2.4.6 was a special case since those were both enterprise releases.